S/SimpleDifferentialOperators/Compat.toml

```diff
@@ -1,2 +1,2 @@
-[0]
+["0-0.4.1"]
 julia = "1.1-1"
```

Comment:
Doesn't require Julia 1.1 anymore?

Reply:
Sorry, not sure I follow.

Reply:
Your last release required at least Julia 1.1. This one seems not to.

Reply:
Probably a mistake. This is our first time using Registrator. I'll fix it.

Reply:
Let me know if it's good now.